### DISCUSSION TITLE
Sampling scheme requirements for different model fits

### DISCUSSION THREAD

**Discussion by peterneher** [dipy/dipy#2613] [Reconstruction]
Hi everyone, can I find information about what sampling scheme (number of b-values, minimum number of directions, etc. ) is required for which fit (e.g. NODDI, CHARMED, ...) somewhere?  

Best,
Peter

**Comment by skoudoro** [comment]
Hi @peterneher,

I do not have a detailed answer to your question but this table can help you: https://dipy.org/documentation/1.5.0/examples_index/#reconstruction
